rel/files/mongooseim.toml

```diff
@@ -62,6 +62,8 @@
   [[listen.http.handlers.mongoose_domain_handler]]
     host = "localhost"
     path = "/api"
+    username = "admin"
+    password = "secret"
```

Comment:
I am not sure if it's good to set up such defaults in the config file... they aren't more secure than having no auth. Maybe we could leave out these two options? This way the user would have to consciously set something here. This would be also more in line with the docs, which docuemnt these two fields as empty by default. I know that this is not equal to them being empty in the default config file... but in this case I think they can be.

Reply:
do we have a way to dynamically start cowboy handlers?